modules/KoreBuild.Tasks/KoreBuild.Tasks.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
What requires this change?

Reply:
I wanted to use `Path.GetRelativePath`, which is not in ns2.0, but is in netcoreapp2.0.